scripts/verify-multiuser-docker.sh

```
@@ -114,6 +118,71 @@ check "alice terminal runs as alice" $?
+  -d '{"username":"alice","password":"alice-password"}'
+
+# Create a Quick Session in alice's HOME so the route does not have to
+# traverse anywhere with restrictive perms.
+session_code="$(curl -s -o "$SESSION_RESP" -w '%{http_code}' -b "$COOKIE_JAR" -c "$COOKIE_JAR" \
+  -X POST "${BASE_URL}/api/sessions" \
+  -H 'Content-Type: application/json' \
+  -d '{"type":"quick","locationPath":"/home/alice"}')"
+echo "  POST /api/sessions (quick, /home/alice) -> HTTP ${session_code}"
+session_id="$(grep -o '"id":"[^"]*"' "$SESSION_RESP" | head -n1 | cut -d'"' -f4)"
+[ "$session_code" = "201" ] && [ -n "$session_id" ]
+check "alice can create a session" $?
```

Shellcheck (0.11.0)

[warning] This $? refers to a condition, not a command. Assign to a variable to avoid it being overwritten.

(SC2319)